get_optee_qemuv8.sh

```diff
@@ -5,11 +5,11 @@
 #
 # [1] https://optee.readthedocs.io/en/latest/building/devices/qemu.html#qemu-v8
 
-ROOT_DIR=${1:-/root/optee_repo_qemu_v8}
+ROOT_DIR=/root/optee_repo_qemu_v8
 set -e
 mkdir -p ${ROOT_DIR}
 cd ${ROOT_DIR}
-repo init -u https://github.com/OP-TEE/manifest.git -m qemu_v8.xml
+repo init -u https://github.com/lorc/optee-manifest.git -m qemu_v8.xml -b zephyr-optee
```

Comment:
are you sure it should be /lorc/optee-manifest.git? "lork"

Reply:
Oops, fixed that. Thank you.